packages/at_secondary_server/lib/src/verb/handler/info_verb_handler.dart

```diff
@@ -55,25 +55,6 @@ class InfoVerbHandler extends AbstractVerbHandler {
           infoMap['apkam_metadata'] = result;
         }
       }
-      infoMap['features'] = [
-        {
-          "name": "noop:",
-          "status": "Beta",
-          "description":
-              "The No-Op verb simply does nothing for the requested number of milliseconds. "
-                  "The requested number of milliseconds may not be greater than 5000. "
-                  "Upon completion, the noop verb sends 'ok' as a response to the client.",
-          "syntax": VerbSyntax.noOp
-        },
-        {
-          "name": "info:",
-          "status": "Beta",
```

Comment:
Do we need to cut out so much, when the only part that needs changing is the status?